bin/init.sh

```diff
@@ -55,9 +47,14 @@ if [ ! -f vendor/envoy-$PROXYVERSION ] ; then
     curl -Lo - https://storage.googleapis.com/istio-build/proxy/envoy-$PROXY.tar.gz | tar xz
     cp usr/local/bin/envoy $ISTIO_GO/vendor/envoy-$PROXYVERSION
     popd
+fi
 
+if [ ! -f $GO_TOP/bin/envoy ] ; then
```

Comment:
If there is an older version envoy.  The latest version will not copied here.   
Some tests (such as mixer/test/cient/...) are just using "envoy" directly.  They will not be using the correct proxy version.

Reply:
Added a rm - we could also do a diff.